BUILD.md

```diff
@@ -91,3 +91,18 @@ python setup.py develop
 
 A lot of variables are available, consider reading [Variables.md](./Variables.md) for more information.
 
+
+# Contribute
```

Comment:
Why not use a [CONTRIBUTING file](https://help.github.com/articles/setting-guidelines-for-repository-contributors/)?